src/main/java/core/basesyntax/service/serviceimpl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    public static final String SEPARATOR = ",";
```

Comment:
Consider encapsulating the constants `SEPARATOR` and `TITLE` better. If they are only used within this class, they should be private.